Unite.Web/Client/src/cohorts/CohortsPage.vue

```diff
@@ -31,11 +31,15 @@
 
   <!-- Page content -->
   <div class="col q-gutter-y-sm">
-    <div class="row">
+    <!-- <div class="row">
```

Comment:
Can this be deleted?

Reply:
We need to leave it, as I don't know yet, how breadcrumbs should be displayed.